### ISSUE TITLE
Unable to run sample

### ISSUE THREAD

**Issue by simanaitis** [joaompneves/tsviz#2]
I tried to run sample but got folowing error:

```
  throw new Error(typeof element + " not supported in " + typeof this);
    ^

    Error: object not supported in object at Module.Element.getElementCollection (C:\Users\Username\Desktop\tsviz-master\bin\ts-elements.js:63:15).
```

Am I doing something wrong or is it bug ?

**Comment by hiigami** [comment]
getting the same error:

/usr/local/lib/node_modules/tsviz/bin/ts-elements.js:64
        throw new Error(typeof element + " not supported in " + typeof this);
        ^

Error: object not supported in object
    at Module.Element.getElementCollection (/usr/local/lib/node_modules/tsviz/bin/ts-elements.js:64:15)
    at Module.getElementCollection (/usr/local/lib/node_modules/tsviz/bin/ts-elements.js:127:54)
    at Module.Element.addElement (/usr/local/lib/node_modules/tsviz/bin/ts-elements.js:60:14)
    at analyseNode (/usr/local/lib/node_modules/tsviz/bin/ts-analyser.js:58:28)
    at /usr/local/lib/node_modules/tsviz/bin/ts-analyser.js:63:56
    at visitEachNode (/usr/local/lib/node_modules/tsviz/node_modules/typescript/lib/typescript.js:7417:30)
    at Object.forEachChild (/usr/local/lib/node_modules/tsviz/node_modules/typescript/lib/typescript.js:7646:21)
    at analyseNode (/usr/local/lib/node_modules/tsviz/bin/ts-analyser.js:63:12)
    at /usr/local/lib/node_modules/tsviz/bin/ts-analyser.js:63:56
    at visitEachNode (/usr/local/lib/node_modules/tsviz/node_modules/typescript/lib/typescript.js:7417:30)

**Comment by joaompneves** [comment]
The problem is related to changes introduced in new typescript version. I'll update the source code soon.

**Comment by joaompneves** [comment]
Fixed in 1cbdf3730eb9b3e46560cf09914f9d62a3bdf101

**Comment by hiigami** [comment]
+1
